packages/components/src/checkbox-control/style.scss

```diff
@@ -5,6 +5,20 @@
 	@include break-small() {
 		--checkbox-input-size: 20px;
 	}
+
+	.components-base-control__help {
+		margin-left: calc(var(--checkbox-input-size) + #{$grid-unit-15});
+	}
+
+	.components-base-control__field {
+		// Emsure label doesn't wrap beneath checkbox
+		display: flex;
+	}
```

Comment:
We can avoid reaching into BaseControl internals here by wrapping our elements in an `<HStack spacing={ 0 } justify="start" alignment="top">` instead.